vulnerabilities/views.py

```diff
@@ -96,17 +104,34 @@ class VulnerabilitySearch(ListView):
     ordering = ["vulnerability_id"]
     paginate_by = PAGE_SIZE
 
+    def get_paginate_by(self, queryset):
```

Comment:
Why provide the queryset as an arg? It does not seems to be used.

Reply:
It’s included to match the expected method signature from Django, which sometimes passes the queryset by default.